src/core/CCFormat.mli

```diff
@@ -84,48 +84,48 @@ val within : string -> string -> 'a printer -> 'a printer
 val map : ('a -> 'b) -> 'b printer -> 'a printer
 
 val vbox : ?i:int -> 'a printer -> 'a printer
-(** Wrap the printer in a vertical box
-    @param i level of indentation within the box (default 0)
+(** Wrap the printer in a vertical box.
+    @param i level of indentation within the box (default 0).
     @since 0.16 *)
 
 val hvbox : ?i:int -> 'a printer -> 'a printer
-(** Wrap the printer in a horizontal/vertical box
-    @param i level of indentation within the box (default 0)
+(** Wrap the printer in a horizontal/vertical box.
+    @param i level of indentation within the box (default 0).
     @since 0.16 *)
 
 val hovbox : ?i:int -> 'a printer -> 'a printer
-(** Wrap the printer in a horizontal or vertical box
-    @param i level of indentation within the box (default 0)
+(** Wrap the printer in a horizontal or vertical box.
+    @param i level of indentation within the box (default 0).
     @since 0.16 *)
 
 val hbox : 'a printer -> 'a printer
-(** Wrap the printer in an horizontal box
+(** Wrap the printer in an horizontal box.
     @since 0.16 *)
 
 val return : ('a, _, _, 'a) format4 -> unit printer
 (** [return "some_format_string"] takes a argument-less format string
     and returns a printer actionable by [()].
     Examples:
-    - [return ",@ "]
-    - [return "@{<Red>and then@}@,"]
-    - [return "@[<v>a@ b@]"]
+    - [return ",@ "].
```

Comment:
I'd rather not put dots here, it'll look like it's part of the code snippet

Reply:
OK. No dots.